gsms.py

```diff
@@ -259,64 +448,73 @@ def main() -> None:
                 src_image = os.path.join(args.shortcut_dir, 'StreamingAssets', name, 'box-art.png')
                 dst_image = os.path.join(args.image_path, f'{name}.png')
 
-                # if src_image exists and dst_image does not exist
-                if os.path.isfile(src_image) and not os.path.isfile(dst_image):
-                    shutil.copy2(src=src_image, dst=dst_image)  # copy2 preserves metadata
-                    print(f'Copied box-art image to: {dst_image}')
-                else:
-                    print(f'No box-art image found at: {src_image}')
+                copy_image(src_image=src_image, dst_image=dst_image)
```

Comment:
Let's only do this when.
`if not args.dry_run:`

Probably requires an update to the options at the top, the args help statement, and the readme file.

Reply:
Will do

Reply:
Seems this wasn't added. Just updated my most recent review.